frontend/src/components/Volunteer/volunteer.jsx

```diff
@@ -0,0 +1,129 @@
+                    required=""
+                    type="email"
+                    className="col-md-10 inputs"
+                  />
+                </div>
+                <br />
+                <div>
+                  <label for="phone" className="col-md-2 block labelFont">
+                    <b>Phone</b>(required)
+                  </label>
+                  <input id="phone" type="text" className="col-md-10 inputs" />
+                </div>
+                <br />
+                <div>
+                  <label for="website" className="col-md-2 block labelFont">
+                    <b>Website</b>
+                  </label>
+                  <input
+                    id="website"
+                    type="text"
+                    className="col-md-10 inputs"
+                  />
+                </div>
+                <br />
+              </Col>
+              <Col xs={12} sm={12} md={12} lg={5}>
+                <h2 className="mb-4">Volunteer!</h2>
+                <p>
+                  Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa
+                  molestias at laudantium, distinctio nulla quo minima delectus
+                  consequuntur aut incidunt ipsam quidem repellat fugiat
+                  consequatur nihil odit quam voluptas earum.Lorem ipsum dolor
+                  sit amet consectetur adipisicing elit. Ipsa molestias at
+                  laudantium, distinctio nulla quo minima delectus consequuntur
+                  aut incidunt ipsam quidem repellat fugiat consequatur nihil
+                  odit quam voluptas earum.Lorem ipsum dolor sit amet
+                  consectetur adipisicing elit. Ipsa molestias at laudantium,
+                  distinctio nulla quo minima delectus consequuntur aut incidunt
+                  ipsam quidem repellat fugiat consequatur nihil odit quam
+                  voluptas earum.Lorem ipsum dolor sit amet consectetur
+                  adipisicing elit. Ipsa molestias at laudantium, distinctio
+                  nulla quo minima delectus consequuntur aut incidunt ipsam
+                  quidem repellat fugiat consequatur nihil odit quam voluptas
+                  earum.Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                  Ipsa molestias at laudantium, distinctio nulla quo minima
+                  delectus consequuntur aut incidunt ipsam quidem repellat
+                  fugiat consequatur nihil odit quam voluptas earum.Lorem ipsum
+                  dolor sit amet consectetur adipisicing elit. Ipsa molestias at
+                  laudantium, distinctio nulla quo minima delectus consequuntur
+                  aut incidunt ipsam quidem repellat fugiat consequatur nihil
+                  odit quam voluptas earum.
+                </p>
+              </Col>
+            </Row>
+            <Row>
+              <Col>
+                <div>
+                  <label for="message" className=" block labelFont">
```

Comment:
yeah as discussed -> lets move this up to the end of the form and add a min-width for the extension